CHANGELOG.md

```diff
@@ -78,7 +78,10 @@ Adding a new version? You'll need three changes:
 
 ## Unreleased
 
-Nothing yet.
+### Changed
+
+- use `req.Clone` improve RoundTrip function.
+  [#5089](https://github.com/Kong/kubernetes-ingress-controller/pull/5089)
```

Comment:
@czeslavo @pangruoran I'd suggest we not put this type of entries to the changelog since the related changes do not bring any user facing changes.

I'd say that constructing such an entry I'd want to answer the question: if I was using KIC, would I care to know about or notice this change? If the answer (which I believe is in this case) I wouldn't add anything here.

Reply:
ok, thanks! Got it.